api/app/src/main/kotlin/packit/clients/GithubUserClient.kt

```diff
@@ -38,14 +37,25 @@ class GithubUserClient(private val config: AppConfig, private val githubBuilder:
         return UserPrincipal.create(user, mutableMapOf())
     }
 
-    fun checkGithubOrgMembership()
+    fun checkGithubMembership()
     {
         checkAuthenticated()
 
-        val userOrgs = ghUser!!.allOrganizations.map {org -> org.login}
-        val inAllowedOrg = allowedOrgs.any {allowed -> userOrgs.contains(allowed)}
+        val userOrg = ghUser!!.allOrganizations.firstOrNull { org -> org.login == config.authGithubAPIOrg }
```

Comment:
also `!!` feels kinda unsafe... is it better to null coalsece? (i.e `ghUser?.allOrganisations`)